tests/testsuite/kubevirtresource.go

```diff
@@ -116,6 +116,7 @@ func AdjustKubeVirtResource() {
 		featuregate.WorkloadEncryptionSEV,
 		featuregate.VMExportGate,
 		featuregate.KubevirtSeccompProfile,
+		featuregate.DecentralizedLiveMigration,
```

Comment:
Can we do this in a follow up PR? That way we'll run all the tests without the FG at least once. Alternatively, we could create a periodic lane that runs without the FG, but I don't think it's really necessary.

Reply:
> Any chance you could have the tests be serial and enable the FG for now? Sorry for the hassle...

Sure...